spec/api/v4/cvd_risk_spec.rb

```diff
@@ -0,0 +1,58 @@
+require "swagger_helper"
+
+describe "CVD Risk V4 API", swagger_doc: "v4/swagger.json" do
+  let(:request_user) { create(:user) }
+  let(:request_facility) { create(:facility, facility_group: request_user.facility.facility_group) }
+  let(:HTTP_X_USER_ID) { request_user.id }
+  let(:HTTP_X_FACILITY_ID) { request_facility.id }
+  let(:Authorization) { "Bearer #{request_user.access_token}" }
+
+  path "/cvd_risks/sync" do
+    post "Syncs cvd_risks data from device to server." do
+      tags "CVD Risk"
+      security [access_token: [], user_id: [], facility_id: []]
+      parameter name: "HTTP_X_USER_ID", in: :header, type: :uuid
+      parameter name: "HTTP_X_FACILITY_ID", in: :header, type: :uuid
+      parameter name: :cvd_risks, in: :body, schema: Api::V4::Schema.cvd_risk_sync_from_user_request
+
+      response "200", "cvd_risks created" do
```

Comment:
```suggestion
      response "200", "cvd risks received" do
```

This would be more consistent with the rest of the documentation and more technically correct, because a sync merge may or may not create a new resource on the server.